go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/openebs/api
```

Comment:
lets have module name as `openebs.io/api` , just change `module openebs.io/api`

Reply:
Chaging the module name you have to make changes in code-gen as well, to generate with `openebs.io` import path

Reply:
As discussed -- will do this later once we migrate our operators.